run/shared/species_database.yml

```diff
@@ -4992,17 +5167,26 @@ STYR:
   FullName: Styrene
   Is_Gas: true
   MW_g: 104.1491
+TDST:
```

Comment:
Should this also include a line `  << : *DSTbinproperties`?

Reply:
Actually after talking with @1Dandan, TDST is only used to get the emitted mass of dust species and not for other processes.  So its entry should be:
```console
TDST:
  # NOTE: TDST is only used to obtain the total emitted mass across
  # all dust bins, and should not be used for other processes.
  Density: 2500.0
  FullName: Total dust aerosol
  MW_g: 29.0
  Is_Aerosol: true
```